app/models/models.rb

```diff
@@ -15,7 +15,10 @@ module QDM
 # FacilityLocation is technically a basetype
 require_relative 'qdm/facility_location'
 
+
 # Tacoma-specific models
+require_relative 'cqm/provider'
+require_relative 'cqm/provider_performance'
```

Comment:
This is a generated file, changes should be made to templates/models_template.rb.erb You can regenerate everything by running `rm app/models/qdm/*.rb && ruby lib/generate_models.rb modelinfo/qdm-modelinfo-5.4.xml data/oids.json`